.github/workflows/buildAndTestWindows.yml

```diff
@@ -71,6 +71,7 @@ jobs:
         shell: pwsh
         run: |
           ./utils/find-vs.ps1
+          ./utils/get-capnp.ps1 Release
```

Comment:
Since this doesn't change, I'd prefer to install it in the image itself so we don't have to recompile it in every build.

Reply:
Seems reasonable to me. What would an alternative place be to test this script? Ideally, I think there should be a place in CI where these utility scripts are tested to ensure that they don't diverge from a working state.